Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
Apparently I aped [@DomT4's practices](https://github.com/Homebrew/homebrew-core/blob/bf8eb02904161e12a6317851c247df51d4f22a4b/Formula/termshare.rb) in #4525 and was [told](https://github.com/Homebrew/homebrew-core/pull/4525#discussion_r77465582) I was wrong. Nothing is reliable 👿